stingray/pulse/search.py

```diff
@@ -549,3 +553,106 @@ def phaseogram(
         additional_info = {}
 
     return phas, binx, biny, additional_info
+
+
+class Pulsar_Periodogram:
```

Comment:
`PulsarPeriodogram` (maybe better without the underscore?) should inherit `StingrayObject`, so that many of the existing methods for I/O and other stuff work out of the box